### ISSUE TITLE
Joplin consuming CPU capacity

### ISSUE THREAD

**Issue by carbonmetrics** [laurent22/joplin#6223]
Upon startup, Joplin runs a process called `/temp/.mount_JoplindDs4R3/@joplinapp-desktop --type=renderer --user -d`.
This consumes ~ 30-40% of CPU capacity, and results in my (noisy) fan kicking in. 
It forces me to close down Joplin. 

## Environment

Joplin version: 2.6.10 
Platform: Ubuntu 20.04.4 LTS x86_64
OS specifics: Xubuntu

Joplin 2.6.10 (prod, linux)

Client ID: 7b2eef949f0948fb8bfab2e8392e9ac8
Sync Version: 3
Profile Version: 41
Keychain Supported: No

Revision: 98fba37

## Steps to reproduce

1. Start Joplin
2. Wait one minute for process to start
3. Close down Joplin because of the fan noise.

## Describe what you expected to happen

Idling at low CPU capacity when Joplin is not used.

## Logfile
[logfile.txt](https://github.com/laurent22/joplin/files/8159150/logfile.txt)

**Comment by roman-r-m** [comment]
Do you have any plugins installed? Try disabling them is you do.

**Comment by laurent22** [comment]
Looks like it's the Untagged plugin again. I wonder if we should disable it for now and until this issue is addressed.

**Comment by roman-r-m** [comment]
Ah, right, didn't see the log.
How can you disable it? The only way I can think of is creating another release

**Comment by laurent22** [comment]
We could temporarily mark it as "obsolete" in the plugin repo, and it won't show up anymore in the search results. Those who have already installed it though would still have it.

To stop it application-side we would indeed need to create a new release, probably with bespoke code to prevent it from loading, but hopefully that won't be necessary.

**Comment by carbonmetrics** [comment]
Yes, it was the Untagged plugin. I have disabled it, problem solved.
